.github/workflows/driver_ci.yaml

```diff
@@ -0,0 +1,200 @@
+         - csiblock-component: controller
+           command-to-run: "./scripts/run_unitests.sh"
+         - csiblock-component: node
+           command-to-run: "make test-xunit-in-container"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: CSI-${{ matrix.csiblock-component }}- Unit testing + coverage
+        run: |
+          mkdir -p build/reports && chmod 777 build/reports
+          $command_to_run `pwd`/build/reports
+        env:
+          command_to_run: ${{ matrix.command-to-run }}
+
+  CSI-deployment-k8s_yamls_validation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: CSI-deployment- k8s yamls validation
+        run: |
+          ./scripts/run_yamlcheck.sh
+  CSI-Build-and-push-images:
+    runs-on: ubuntu-latest
+    needs:
+      - CSI-controller-static_code_analysis
+      - Unit_testing_coverage
+      - CSI-deployment-k8s_yamls_validation
+    strategy:
+      matrix:
+        image_type: ['node', 'controller']
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1.3.0
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: 'csiblock1'
+          password: '${{ secrets.CSIBLOCK_DOCKERHUB_PASSWORD }}'
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-${{ matrix.image_type }}
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Create github branch parameter
+        id: tested_github_branch
+        run: |
+          docker_image_branch_tag=`echo $CI_ACTION_REF_NAME| sed 's|/|.|g'`
+          echo "::set-output name=docker_image_branch_tag::${docker_image_branch_tag}"
+      - name: Build and push ${{ matrix.image_type }}
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          platforms: linux/amd64,linux/ppc64le,linux/s390x
+          push: true
+          tags: |
+            csiblock1/ibm-block-csi-${{ matrix.image_type }}:${{ steps.tested_github_branch.outputs.docker_image_branch_tag }}_${{ github.run_number }}
+            csiblock1/ibm-block-csi-${{ matrix.image_type }}:${{ steps.tested_github_branch.outputs.docker_image_branch_tag }}
+            csiblock1/ibm-block-csi-${{ matrix.image_type }}:latest
+          file: Dockerfile-csi-${{ matrix.image_type }}
+          cache-from: type=local,src=/tmp/.buildx-${{ matrix.image_type }}
+          cache-to: type=local,dest=/tmp/.buildx-new-${{ matrix.image_type }}
+        # Temp fix
+        # https://github.com/docker/build-push-action/issues/252
+        # https://github.com/moby/buildkit/issues/1896
```

Comment:
What is it? Can we create a ticket with more information for that?

Reply:
We need a ticket for that (otherwise, it will be forgotten).

Reply:
I will open and write the ticket number in the code

Reply:
Great

Reply:
done